packages/reactotron-react-native/package.json

```diff
@@ -69,6 +69,7 @@
   },
   "dependencies": {
     "mitt": "^1.1.2",
+    "pako": "1",
```

Comment:
Are we sure RN doesn't ship with something like this already? I get the need for the lib but I just want to make sure we are not adding it just to add it.

Reply:
As far as I know, RN doesn't ship with a zlib library, and pako seems to be the gold standard (3 million downloads last week).